### PR TITLE
Add/foundry vtt

### PR DESCRIPTION
**Pull request by Gorian**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your server use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

**Comment by Software-Noob** [comment]
Please fill the New Server Submissions template, and add the egg to corresponding READMEs to find it. That is, the main readme and the one in game_eggs

**Comment by Gorian** [comment]
Ah, I didn't realize that "New Server Submissions" meant "eggs" - I thought it was for something else.

**Comment by Gorian** [comment]
I'll have to wait until I'm home to make the changes to the script. Thanks for the tip!

File: game_eggs/FoundryVTT/egg-FoundryVTT.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
...zip\"\r\n\r\nmain() {\r\n  apt update\r\n  apt install -y unzip\r\n\r\n  printf \"\\nBuilding directory structure...\\n\"\r\n  mkdir -p \"${DIR_ROOT}\/data\/Config\"\r\n  mkdir -p \"${DIR_ROOT}\/app\"\r\n  # shellcheck disable=SC2164\r\n  cd \"${DIR_APP}\"\r\n  printf \"\\nDownloading FoundryVTT files...\\n\"\r\n  wget \"${TIMED_URL}\" -O \"${ZIP_FILE_NAME}\"\r\n  printf \"\\nunzipping FoundryVTT files...\\n\"\r\n  unzip \"${ZIP_FILE_NAME}\" -d \"${DIR_APP}\"\r\n  rm \"${ZIP_FILE_NAME}\"\r\n\r\n  printf \"\\nGenerating default configuration...\\n\"\r\n  cat <<EOF >\"${DIR_DATA}\/Config\/options.json\"\r\n{\r\n  \"port\": 30000,\r\n  \"upnp\": false,\r\n  \"fullscreen\": false,\r\n  \"hostname\": null,\r\n  \"localHostname\": null,\r\n  \"routePrefix\": null,\r\n  \"sslCert\": null,\r\n  \"sslKey\": null,\r\n  \"awsConfig\": null,\r\n  \"dataPath\": \"\/home\/container\/data\",\r\n  \"passwordSalt\": null,\r\n  \"proxySSL\": false,\r\n  \"proxyPort\": null,\r\n  \"minifyStaticFiles\": true,\r\n  \"updateChannel\": \"release\",\r\n  \"language\": \"en.core\",\r\n  \"upnpLeaseDuration\": null,\r\n  \"world\": null\r\n} \r\nEOF\r\n  printf \"Installation Done.\\n\"\r\n}\r\nmain \"@\"",
```
Comment (parkervcp) [inline game_eggs/FoundryVTT/egg-FoundryVTT.json:25]: If you made this on windows it may have added the \r and that may cause issues.

Comment (Gorian) [reply game_eggs/FoundryVTT/egg-FoundryVTT.json:25]: huh, I thought that I set all my editors to use linux newline endings. I'll double-check this when I have a chance. thanks!

**Comment by sircurse** [comment]
I would love to have FroundtryVTT eggs as well, what is missing to have this request ? Is there anything I can help?

File: game_eggs/FoundryVTT/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Foundry VTT
+Foundry VTT is a standalone application built for experiencing multiplayer tabletop RPGs using a feature-rich and modern self-hosted application where your players connect directly through the browser.
+
+# Installation
+Foundry requires a license. In order to use this egg, you will need to purchase a foundry license, select the linux platform from your profile on the website, and then paste the "Timed URL" into the variable when seting up the server.
```
Comment (parkervcp) [inline game_eggs/FoundryVTT/README.md:5]: Can we add a link to their site here?

Comment (sircurse) [reply game_eggs/FoundryVTT/README.md:5]: https://foundryvtt.com/

Comment (parkervcp) [reply game_eggs/FoundryVTT/README.md:5]: I meant to ask if you could add it to the readme.

Comment (Gorian) [reply game_eggs/FoundryVTT/README.md:5]: > I meant to ask if you could add it to the readme.

done

**Comment by tempusthales** [comment]
I've been running this egg for the past couple of months without any issues experienced.

**Review by tempusthales — APPROVED** [review]

**Review by tempusthales — APPROVED** [review]
I have been running this build of Foundry VTT for the past couple of months without any issues experienced.  Working as intended.

**Review by parkervcp — APPROVED** [review]

**Pull request merged**